fix: typos in some webhook verify policies (#1587)
ngrok/ngrok-docs: docs/integrations/circleci/webhooks.mdx

> 1. Create a traffic policy file named `circleci_policy.yml`, replacing `{your secret token}` with the value you have provided to the **Secret token** field during your webhook registration (See [Integrate ngrok and CircleCI](#setup-webhook)):

Reviewer: ```suggestion
1. Create a traffic policy file named `circleci_policy.yml`, replacing `your_secret_token` with the value you've provided to the **Secret token** field during your webhook registration (see [Integrate ngrok and CircleCI](#setup-webhook)):
```
